project.py

```diff
@@ -114,6 +134,79 @@ def create_trackbars(self):
         cv.createTrackbar('lo', 'result', self.lo, 255, self.trackbar_callback)
         cv.createTrackbar('hi', 'result', self.hi, 255, self.trackbar_callback)
 
+    def check_change(self, image, mask):
+        imageOld = cv.imread("slide"+repr(self.numberOfSlides)+".png",
```

Comment:
@Foxiy I haven't read this through yet but I do not see why should we read the last image from filesystem, rather than hold it in memory. Is there some good reason for that?